job/envs.sh

```diff
@@ -10,3 +10,5 @@
   return 1
 }
 <% }); %>
+
+export SUBSCRIPTION_PRIVATE_KEY="$BUILD_DIR/secrets/00_sub"
```

Comment:
- isnt this value already being parsed somewhere in js code? if it is, then lets use that instead of assembling this again